.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need to add here some basic configuration like:
```
{
  "trailingComma": "es5",
  "tabWidth": 4,
  "semi": false,
  "singleQuote": true
}
```
?

Reply:
I left it empty because the default value seems already pretty aligned with our current convention. I ran `yarn prettier --write "packages/client/src/**/*.js"` and created second commit with updated files.

Reply:
So how does it work, do we need to run this command before commit or it's formatting the file on save?
Just thinking from the perspective if we need to add this command to scripts in package.json